openstates/md/bills.py

```diff
@@ -165,10 +170,132 @@ def parse_bill_votes(self, doc, bill):
                 vote.add_source(href)
                 yield vote
 
+    def parse_bill_votes_new(self, doc, bill):
+        elems = doc.xpath('//a')
+
+        # MD has a habit of listing votes twice
+        seen_votes = set()
+
+        for elem in elems:
+            href = elem.get('href')
+            if (href and "votes" in href and href.endswith('pdf') and
+                    ("Senate" in href or "House" in href) and href not in seen_votes):
+                seen_votes.add(href)
+                vote = self.parse_vote_pdf(href, bill)
+                vote.add_source(href)
+                yield vote
+
+    def parse_vote_pdf(self, vote_url, bill):
+
+        filename, response = self.urlretrieve(vote_url)
+
+        text = convert_pdf(filename, type='text').decode()
+        lines = text.splitlines()
+
+        if 'senate' in vote_url:
+            chamber = 'upper'
+        else:
+            chamber = 'lower'
+
+        date_string = lines[0].split('Date:')[1].strip()
```

Comment:
Can you ensure that you're getting `Calendar Date` and not `Legislative Date`? Seems like the `lines[0]` should take care of that, but just in case. Legislative dates are very funky.

Reply:
I can run manual tests, and confirm if it is picking legislative date anywhere, other than that do you think I have to make any change in code as well?

Reply:
More I can check for existence of word ```calendar``` in ```lines[0].split('Date:')[0]```, will it do?

Reply:
Yeah, can you `split('Calendar Date:')`, or something like that?

Reply:
Totally